Clients/Xamarin.Interactive.Client.Mac/AppDelegate.cs

```diff
@@ -172,17 +175,35 @@ public void InstallCommandLineTool (NSObject sender)
                     scriptPath,
                     mode: 493 /*0755*/);
 
-                await authOpen.WriteAsync (h => h.WriteData (NSData.FromString (
+                await scriptAuthOpen.WriteAsync (h => h.WriteData (NSData.FromString (
                     "#!/bin/bash\n" +
                     "\"" + NSBundle.MainBundle.ExecutablePath + "\" cli \"$@\""
                 )));
 
+                const string pathsDPath = "/etc/paths.d/workbooks";
+                var pathsDAuthOpen = await Security.AuthOpen.PreauthorizeAsync (
+                    new XamCore.Security.AuthorizationEnvironment {
+                        Prompt = Catalog.Format (Catalog.GetString (
+                            "{0} would like to install a path helper to " +
+                            "support terminal usage: {1}",
+                            comment: "{0} is the app name; {1} is a file path"),
+                            ClientInfo.FullProductName,
+                            pathsDPath),
+                        AddToSharedCredentialPool = true
+                    },
+                    pathsDPath,
+                    mode: 420 /* 0644 */);
+
+                await pathsDAuthOpen.WriteAsync (h => h.WriteData (NSData.FromString (
+                    Path.GetDirectoryName (NSBundle.MainBundle.ExecutablePath)
+                )));
+
                 new NSAlert {
                     AlertStyle = NSAlertStyle.Informational,
                     MessageText = Catalog.GetString ("Installation Complete"),
                     InformativeText = Catalog.Format (Catalog.GetString (
                         "The '{0}' tool has been installed to {1}. " +
-                        "Ensure that {1} is in your PATH.",
+                        "Close and reopen your terminal to ensure that it is in PATH.",
```

Comment:
Since you no longer use `{0}` or `{1}`, should you remove them and the translator comment? Or are you leaving them in case we want to update the string in the future?

Reply:
Huh? They're both used in the earlier part of the string?

Reply:
Of course they are. Hold on, sorry, gotta answer that. Nice chatting with you!